frontend/packages/core/src/index.tsx

```diff
@@ -1,6 +1,8 @@
 import { Grid } from "@material-ui/core";
 
 import { userId } from "./AppLayout/user";
+import LinearTimeline from "./Charts/linearTimeline";
```

Comment:
We should probably just have an index file in the charts folder that exports all these and can be re-exported here.